.ci/semaphore.tests.sh

```diff
@@ -1,6 +1,6 @@
 set -e -x
 
-pip install coala-bears
+pip install git+https://github.com/coala/coala-bears.git@master
```

Comment:
Where is coala? :/

Reply:
It's a dependency of coala-bears. That way we let coala-bears manage its compatible coala version bears are written for.

Reply:
Ah, okay. :+1: